### PR TITLE
[DUOS-806][risk=low] Specify ignore file on command line

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadinstitute.atlassian.net/browse/DUOS-806
See also DataBiosphere/consent#785 for a more comprehensive fix

Temporary workaround for addressing a known, unfixed vulnerability.

----

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/consent-ontology/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/consent-ontology/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
- [ ] I've updated Swagger to reflect any API changes

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -91,7 +91,7 @@ function docker_cmd()
         docker build -t $DOCKERHUB_REGISTRY:${HASH_TAG} .
 
         echo "scanning docker image..."
-        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
+        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL --ignorefile .trivyignore "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```
Comment (dinvlad) [inline build.sh]:
```suggestion
        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ -v "$PWD/.trivyignore":/.trivyignore:ro aquasec/trivy --exit-code 1 --severity CRITICAL --ignorefile /.trivyignore "$DOCKERHUB_REGISTRY":"${HASH_TAG}"
```
this assumes that `docker run` command runs from the repo root - could you confirm?

Comment (rushtong) [reply build.sh]: I don't know how jenkins runs these ... will add the volume mount though.

**Review by dinvlad — APPROVED** [review]

File: build.sh (hunk 1 above)
Comment (rushtong) [reply build.sh]: Actually, I do think the command runs from the repo root. This is the shell in jenkins:
```
#!/bin/bash
export GIT_COMMIT=$(git log -n 1 --pretty=format:%H)
./jenkins/jenkins_build.sh
```

**Comment by rushtong** [comment]
@dinvlad Thank you!

**Pull request merged**